CONTRIBUTING.md

```diff
@@ -1,20 +1,20 @@
 # Contributing Guide
+
+If you want to help develop this project, and you don't know where to start, consider:
```

Comment:
We could add a line that we welcome contributions. Also we can encourage asking questions on discord before starting a work